src/Parsers/ASTCreateNamedCollectionQuery.h

```diff
@@ -14,6 +14,7 @@ class ASTCreateNamedCollectionQuery : public IAST, public ASTQueryWithOnCluster
     std::string collection_name;
     SettingsChanges changes;
     bool if_not_exists = false;
+    SettingsChanges overridability;
```

Comment:
is there a reason to use `SettingsChanges`? it is a vector, but seems it is better to use `std::unordered_map`